core/src/main/kotlin/com/exactpro/th2/read/db/core/MessageLoader.kt

```diff
@@ -16,17 +16,20 @@
 
 package com.exactpro.th2.read.db.core
 
+import java.time.Instant
+
 fun interface MessageLoader {
     /**
      * Loads the most recent message from the Cradle produced by data source with [dataSourceId]
-     * that has a specified set of [properties] in its metadata
+     * that message timestamp is after or equal [horizon] and has a specified set of [properties] in its metadata
      * @param dataSourceId is used for session alias determination related to the data source
+     * @param horizon where messages are actual
```

Comment:
What does `actual` mean in this case? I think we should rephrase this because `actual` can be interpreted in many ways. Especially, in pair with the `horizon` parameter name.
I would suggest something like `boundary` or `lowerBoundary` or `afterTimestamp` for the parameter name. The description could be like `the time boundary for message loading. Only messages with a timestamp greater or equal to this value will be loaded if exist. Must be lower than the current time if set`